jbpm-designer-backend/src/main/java/org/jbpm/designer/bpmn2/validation/BPMN2SyntaxChecker.java

```diff
@@ -96,15 +97,15 @@ public void checkSyntax() {
                             foundPackageName = true;
                             pname = (String) entry.getValue();
                             if(isEmpty(pname)) {
-                                addError(defaultResourceId, new ValidationSyntaxError(process, BPMN2_TYPE,  "Process has no package name."));
+                                addError(defaultResourceId, new ValidationSyntaxError(process, BPMN2_TYPE, SyntaxCheckerErrors.PROCESS_HAS_NO_PACKAGE_NAME));
                             }
                             if(!isValidPackageName(pname)) {
                                 addError(defaultResourceId, new ValidationSyntaxError(process, BPMN2_TYPE,  "Package name contains invalid characters."));
```

Comment:
I can do it. It was my original plan to extract all constants from BPMN2SyntaxChecker but then I decided to extract only constants which I use in tests. But I can extract all constants right now. I will amend PR soon.